x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -48,14 +63,57 @@ function useServicesFetcher() {
               end,
             },
           },
+        }).then((response) => {
+          return {
+            requestId: uuid(),
+            mainStatistics: response,
+          };
         });
       }
     },
-    [environment, kuery, start, end]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [
+      environment,
+      kuery,
+      start,
+      end,
+      // not used, but needed to trigger an update when offset is changed either manually by user or when time range is changed
```

Comment:
the way that I've implemented the `offset` parameter in other APIs is that it will automatically offset the queried time range, but also the buckets - is that an option here as well?